### PR TITLE
Added Italian translation

### PR DESCRIPTION
**Pull request by Romloader**
### Added italian translation to the app

Updated *`app/src/main/res/values-it/strings.xml`*

### CODE REVIEW

**Comment by pro776** [comment]
The app stops working once I exit the app. No idea why?
I'm using a Asus Zenfone Max Pro M1 6/64, Android Oreo

**Comment by sleepii-ux** [comment]
I can use the scanner to open sesame shortcuts search, but then it breaks and won't scan again until I disable then re-enable the service

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
+
+    <string name="app_name">Fingerprint Quick Action</string>
+
+    <string name="xposed_description">Esegue azioni veloci (gestures) con un tocco o trascinamento sul sensore di impronte. L\'attivazione del modulo Xposed rende l\'app più stabile. (Nougat+ non supportato al momento)</string>
+
+    <!-- FPQAService -->
+    <string name="toast_notify_on_error">Fingerprint Quick Action: Errore\n%1$s</string>
```
Comment (Gr307496) [inline app/src/main/res/values-it/strings.xml:9]: Adam.green3009@gmail.com

**Review by Gr307496 — APPROVED** [review]

**Comment by Romloader** [comment]
Closing this because the project is in an archived state

**Pull request closed without merging**